build.cake

```diff
@@ -766,12 +766,31 @@ Task("PackageZip")
         Zip(currentImageDir, File(ZIP_PACKAGE_SL), zipFiles);
     });
 
-Task("PackageMsi")
+
+Task("PackageRunnersMsi")
+    .IsDependentOn("CreateImage")
+    .WithCriteria(IsRunningOnWindows)
+    .Does(() =>
+    {
+        MSBuild("runners-msi/runners/nunit.wixproj", new MSBuildSettings()
+            .WithTarget("Rebuild")
+            .SetConfiguration(configuration)
+            .WithProperty("PackageVersion", packageVersion)
+            .WithProperty("DisplayVersion", version)
+            .WithProperty("OutDir", PACKAGE_DIR)
+            .WithProperty("InstallImage", IMAGE_DIR + "NUnit-" + packageVersion)
+            .SetMSBuildPlatform(MSBuildPlatform.x86)
+            .SetNodeReuse(false)
+        );
+    });
+
+
+Task("PackageFrameworkMsi")
```

Comment:
Makes sense, we can eventually have it as an artefact for the console/engine repo and not the framework? Will sort it tomorrow - it's getting on a bit UK time!